src/shared-components/dropdown/mobileDropdown.tsx

```diff
@@ -4,9 +4,21 @@ import PropTypes from 'prop-types';
 import ChevronIcon from '../../svgs/icons/chevron-icon.svg';
 import { DropdownContainer, dropdownInputStyle, IconContainer } from './style';
 
+import { OptionType } from './index';
+
+type MobileDropdownProps = {
+  value: string | null;
```

Comment:
Should we loosen this type to respect the intent of `value: PropTypes.any`? I think `string | number | null` would cover all bases, and we could perhaps change `value: PropTypes.any` to `value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])`.